YtFlowApp/Pages/QrCodePage.xaml.cs

```diff
@@ -0,0 +1,248 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using System.Threading.Tasks;
+using Windows.Devices.Enumeration;
+using Windows.Foundation;
+using Windows.Graphics.Imaging;
+using Windows.Media.Capture;
+using Windows.Media.Devices;
+using Windows.Media.MediaProperties;
+using Windows.Storage.Streams;
+using Windows.UI.Core;
+using Windows.UI.Popups;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media.Imaging;
+using Windows.UI.Xaml.Navigation;
+using ZXing;
+
+// https://go.microsoft.com/fwlink/?LinkId=234238 上介绍了“空白页”项模板
+
+namespace YtFlow.App.Pages
+{
+    /// <summary>
+    /// 可用于自身或导航至 Frame 内部的空白页。
+    /// </summary>
+    public sealed partial class QrCodePage : Page
+    {
+        private Result result;
+        private readonly MediaCapture mediaCapture = new MediaCapture();
+        private DispatcherTimer timer;
+        private bool isBusy;
+
+        public QrCodePage()
+        {
+            this.InitializeComponent();
+        }
+
+        protected override void OnNavigatedTo(NavigationEventArgs e)
+        {
+            InitVideoCapture();
+            InitVideoTimer();
+        }
+
+        protected async override void OnNavigatedFrom(NavigationEventArgs e)
+        {
+            timer.Stop();
+            if (isBusy)
+            {
+                await mediaCapture.StopPreviewAsync();
+            }
+            isBusy = false;
+        }
+
+        private void InitVideoTimer()
+        {
+            timer = new DispatcherTimer();
```

Comment:
Try to stick with a single instance instead of initializing a `DispatcherTimer` every time.

Reply:
Sloved